modules/standard/FileSystem.chpl

```diff
@@ -27,7 +27,7 @@ use Error;
    Note: this is not safe within a parallel context.  A chdir call in one task
    will affect the current working directory of all tasks for that locale.
 */
-proc locale.chdir(out err: syserr, name: string) {
+proc locale.chdir(out error: syserr, name: string) {
```

Comment:
The variable passed doesn't have to have the same name as the argument. We can fix that later if you want - I just wanted the public interface to be set for code freeze. Still waiting on tests...